### PR TITLE
Updating for v4 release

### PR DESCRIPTION
**Pull request by nimmaj**
I believe this is the correct (0.7.10) version of quasar that is consistent with corda v4.  The current version is 0.7.6 which is quite old.  Thanks.

### CODE REVIEW

**Comment by anthonykeenan** [comment]
It's also pointing to the `4.0-SNAPSHOT` and an old version of the gradle plugins.
```
        corda_release_version = '4.0-SNAPSHOT'
        corda_gradle_plugins_version = '4.0.37'
```
Would you possibly mind looking at the `samples` repo which is @chrisr3 has spent a lot of time getting on the correct versions of things and updating what you can. And then doing the same to the `cordapp-template-java` repo?

File: workflows/build.gradle [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 apply plugin: 'net.corda.plugins.quasar-utils'
 
 cordapp {
-    targetPlatformVersion 4
-    minimumPlatformVersion 4
+    targetPlatformVersion corda_platform_version.toInteger()
+    minimumPlatformVersion corda_platform_version.toInteger()
     workflow {
-        name "Template Flows"
+        name "Tempmlate Flows"
```
Comment (anthonykeenan) [inline workflows/build.gradle]: Template

**Review by anthonykeenan — CHANGES_REQUESTED** [review]
Just one typo - otherwise looks good. Thanks!

File: workflows/build.gradle (hunk 1 above)
Comment (nimmaj) [reply workflows/build.gradle]: good spot - apols - fixed

**Review by anthonykeenan — APPROVED** [review]

**Pull request merged**